codeqa.tex

```diff
@@ -36,9 +36,8 @@ \subsubsection{Coding Standards}
 Firstly, we disagree with the whitespace rules concerning binary operators and therefore disable this test.
 Secondly, we feel that an 80 character line limit is too short and we continue to use a 110 character limit for code.
 We did experiment with switching to 80 characters for one package but did not like the result.
-For documentation and comments we  adopted the 75  character limit of Numpydoc.\cite{numpydoc}
-PEP\,8 does specify their line lengths should be different than for code, but at this time there is no tooling in place to warn of non-compliance.
-We have written and submitted a patch to the \texttt{pycodestyle} tool to implement a check for this, which has yet to be  accepted.
+For documentation and comments we have adopted a 79 character limit as a compromise between the 72 characters specified in PEP\,8 and our standard code limit.
```

Comment:
What do you think about changing "documentation" to "documentation strings" / "docstrings" to be more specific?